server/src/endpoints/obelisk/addObeliskNote.ts

```diff
@@ -0,0 +1,38 @@
+import DB from '../../redis'
+import { User } from '../../user'
+import { AuthenticatedEndpointFunction, LogFn } from '../../endpoint'
+
+const addObeliskNote: AuthenticatedEndpointFunction = async (user: User, inputs: any, log: LogFn) => {
+  const message = inputs.message
+  const id = inputs.id
+  if (!message || !id) {
+    return {
+      httpResponse: {
+        status: 500,
+        body: 'Include a post-it message and an ID!'
+      }
+    }
+  }
+
+  await DB.addRoomNote('obelisk', { id, message, authorId: user.id })
```

Comment:
I would be happier if we had a constant somewhere to represent the `obelisk` roomId everywhere I hardcode it, but sigh, it's two days before the conf.